packages/rum-core/src/domain/rumEventsCollection/error/errorCollection.ts

```diff
@@ -85,13 +85,6 @@ function processError(
     error: {
       id: generateUUID(),
       message: error.message,
-      resource: error.resource
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Could we also clean corresponding `rawRumErrorEvent` type?

Reply:
Indeed